python/GangaTest/__init__.py

```diff
@@ -5,6 +5,8 @@ def getEnvironment(c = None):
     return {}
 
 def loadPlugins( config = None ):
+    import Lib.GListApp
```

Comment:
Quite possibly, but at present, this seemed like the best place. If we want to move the testing classes elsewhere that's fine but at least they're all in one place so they can be moved together,